<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HarmonizeDialog</class>
 <widget class="QDialog" name="HarmonizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit geometry</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-fullscreen">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Source type</string>
     </property>
     <property name="buddy">
      <cstring>sourceCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="sourceCombo">
     <item>
      <property name="text">
       <string>Lines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Points</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="x1Label">
     <property name="text">
      <string>X1 index</string>
     </property>
     <property name="buddy">
      <cstring>x1Spin</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="x1Spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="y1Label">
     <property name="text">
      <string>Y1 index</string>
     </property>
     <property name="buddy">
      <cstring>y1Spin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="y1Spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="x2Label">
     <property name="text">
      <string>X2 index</string>
     </property>
     <property name="buddy">
      <cstring>x2Spin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="x2Spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="y2Label">
     <property name="text">
      <string>Y2 index</string>
     </property>
     <property name="buddy">
      <cstring>y2Spin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="y2Spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="distanceLabel">
     <property name="text">
      <string>Distance</string>
     </property>
     <property name="buddy">
      <cstring>distanceSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="distanceSpin">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lengthLabel">
     <property name="text">
      <string>Length</string>
     </property>
     <property name="buddy">
      <cstring>lengthSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="lengthSpin">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="endingsCheck">
       <property name="text">
        <string>Endings only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceCombo</tabstop>
  <tabstop>x1Spin</tabstop>
  <tabstop>y1Spin</tabstop>
  <tabstop>x2Spin</tabstop>
  <tabstop>y2Spin</tabstop>
  <tabstop>distanceSpin</tabstop>
  <tabstop>lengthSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x1Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y1Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x2Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y2Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>fitParametersChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>HarmonizeDialog</receiver>
   <slot>sourceTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fitParametersChanged()</slot>
  <slot>openButtonClicked()</slot>
  <slot>sourceTypeChanged(int)</slot>
 </slots>
</ui>
